### DISCUSSION TITLE
Question About Jet Fan Simulation in FDS – Jet Decay and Recirculation

### DISCUSSION THREAD

**Discussion by Z-0815** [firemodels/fds#13923] [Q&A]
Hello,
I am working on a simulation of jet fans in a closed room using FDS. My issue is that the airflow jet in my simulations stretches far too long and lacks the recirculations seen in ANSYS Fluent. Additionally, the jet decay is not as prominent compared to Fluent simulations. 

![32_0058](https://github.com/user-attachments/assets/1d81c57f-cec0-47d7-b5ec-b444f193d8bc)

The conditions at the jet outlet are already well-resolved with a mesh resolution of h/4 at the jet and h/2 in the surrounding area. My main concern lies with the far-field flow behavior, where the jet continues to extend without the expected decay or recirculation patterns.

Does anyone have experience addressing this issue or suggestions for better modeling of the far-field jet decay and recirculations in FDS?

Thank you for your help, and I wish everyone a Merry Christmas!

**Comment by johodges** [comment]
Do you have a comparison with the Fluent simulations and the FDS input file? It is hard to comment on the differences without being able to see them. What is the exit velocity of your jet fan? The colorbar above is clipped at 1.1 m/s but it looks like that is pretty far below your maximum. We are typically looking at jet fans on the order of 10 m/s in tunnel applications. Thunderhead has done a lot of work on jet fans in FDS and has a pretty good write up here (https://support.thunderheadeng.com/tutorials/pyrosim/modeling-jet-fans/). @bwklein may have more comments on the analysis there.

**Reply by Z-0815** [reply]
Thank you for your response!

The exit velocity of the jet fan in our simulation is approximately 20.27 m/s, matching the intended value of around 20.2 m/s. The primary issue, however, is with the decay of the jet downstream. Based on expectations, the velocity should drop to below 1 m/s at a distance of about 35-45 m from the outlet. In the current simulation, this decay does not occur, and the jet maintains higher velocities over a significantly longer range.

I have reviewed the Thunderhead documentation on jet fans and used their tips to construct the FDS input file. Despite this, the results still show discrepancies in the far-field jet behavior. I would appreciate any additional guidance or suggestions on ensuring proper jet decay, especially in relation to turbulence modeling, boundary conditions, or mesh setup.

Thanks in advance for your help and tips!

**Reply by Z-0815** [reply]
Here is my input file, maybe you have some more ideas.
[32_0058.txt](https://github.com/user-attachments/files/18211130/32_0058.txt)

**Reply by johodges** [reply]
I have it running on our cluster.

**Reply by Z-0815** [reply]
Thank you! I am very eager to see the results and hear your assessment.

**Reply by bwklein** [reply]
@johodges I wonder if using something like `MASS_FLUX_VAR` for HVAC surfaces would help bring a bit more disturbance to the flow and initiate a decay in velocity profile earlier.  I was thinking about that or something like the SEM but this is not allowed for HVAC vents.
In the study on our website, the flow field didn't fall within the bounds of measurements until we got down to ~30-12 mm cell sizes.

@Z-0815 I am running the simulation now where I doubled the resolution in mesh 1 down to 0.0394 m cell size.  This gives us 8 cells across the width of the vent.

I also changed the MPI_PROCESS assignments, to group smaller meshes together to get closer to the cell count of Mesh 1 for each process.

[fan_test.fds.txt](https://github.com/user-attachments/files/18214340/fan_test.fds.txt)

**Reply by bwklein** [reply]
Within the 1st second... 

![fan_test-Default](https://github.com/user-attachments/assets/46412170-896f-4ee8-802f-304812a82644)

**Reply by johodges** [reply]
It doesn't look like it would be too hard to add random noise to an HVAC vent, but I would want @drjfloyd to confirm. CALC_HVAC_BC stores the mdot in B1%M_DOT_G_PP_ADJUST array similar to the other solid boundaries in CALCULATE_ZZ_F. It may be as simple as copying lines 1280:1293 to the end of CALC_HVAC_BC.

**Reply by drjfloyd** [reply]
I think that would break mass conservation. Those lines operate on each wall cell independently and do not appear to do any aggregation to ensure that the total flux over all wall cells is the input flux. Over time those errors will average out, but HVAC relies on strict mass conservation in the network.

**Reply by Z-0815** [reply]
@bwklein, I have started running your input file on my system as well (It will likely run overnight) and am very interested to see the results. Thank you for sharing it!

**Reply by bwklein** [reply]
@Z-0815 run about 4.2 seconds and this is what I am seeing.

![fan_test-Default_4 2s](https://github.com/user-attachments/assets/4e466724-5479-42f7-a33f-c3a91c0417b2)

**Reply by Z-0815** [reply]
@bwklein after approx. 8.3 s. I am curious to see how the jet flow current behaves at greater distances.
![image](https://github.com/user-attachments/assets/aba70ee9-65b8-4e63-991a-c0633595bdc1)

**Reply by drjfloyd** [reply]
@johodges I thought of a way to use that block of code that ensures we conserve mass.  In HVAC for each MPI process we get the area of each HVAC vent on the process and the sum of the boundary conditions next to each HVAC vent, pull those back to the first process to do a weighted average boundary condition, do the HVAC solve, and then push all the mass flows back out to the other processes for use as the wall boundary condition. We could in the first step generate the random numbers for each wall cell and store them in the wall type and sum the weights for each vent like we do for the HVAC inlet conditions. Then in the second step we can sum all the MPI vent weight sums them and invert the sum. Then in the final step when we push mass flows we also push that inverted sum. Then the mass flow we use for each wall cell would be the wall cell weight from the first step times the duct mass flow times the inverted sum. This would make the total masss flow correct.

**Reply by zivbk1** [reply]
@drjfloyd I think a little bit of asymmetry at the HVAC outlet vents will help develop the turbulence a bit earlier in the stream and maybe help match experiment a bit better at more coarse resolutions.

**Reply by johodges** [reply]
@drjfloyd I think that would be helpful as well. I can test it out if you want to make the changes. While you are looking at that, it might be worth seeing if the same plumbing would allow the synthetic eddy method to be applied on the HVAC vents.

**Reply by johodges** [reply]
@Z-0815 this is what I am seeing at the 100 second mark with a 10 second average. What version of FDS were you using? The velocity decay looks reasonable to me and is in line with what you expected in the original post. However, as the other posts have mentioned in this thread, we don't have any specific experimental data to demonstrate whether these results are aligning with experiments.

![image](https://github.com/user-attachments/assets/6001d239-6413-415d-a9d3-c0f1a834089a)
![image](https://github.com/user-attachments/assets/54be12da-11ec-4723-925b-3b1b239d6eef)

**Reply by drjfloyd** [reply]
@johodges I don't think SEM will be compatible.  SEM generates eddies. It is possible that the net flow that results would be in the opposite direction of the required flow.

**Reply by Z-0815** [reply]
@johodges I am using FDS version 6.9.1-0. If the jet flow stops further development and exhibits a decay as shown in your results, it aligns closely with my initial expectations. However, based on simulations with coarser meshes, I have observed that the jet flow tends to grow longer after 100 seconds, which suggests the flow might continue to evolve over extended simulation times.

Additionally, one aspect I am concerned about is the lack of recirculating flows on the sides of the jet. In theory, such recirculations should develop on the left and right, but it might be challenging to capture them accurately, especially since there is airflow coming from the back as well.

Would you recommend evaluating such long simulation times to observe these phenomena, or should the throw distance and other features ideally be assessed at earlier stages of the simulation? Any advice on how to better capture the expected recirculation zones would also be greatly

**Reply by drjfloyd** [reply]
Since this test case is just blowing air (i.e. no smoke that we need to conserve through the jet), I was able to test the idea of using the VEL_RMS by changing the VENTs to an exhaust SURF and a supply SURF with the same flows. On the supply I added VEL_RMS. This did not work well. I think the main issue is the noise is purely random unlike SEM which actually creates eddies. However the SEM method needs sufficient resolution over the VENT for there to be multiple eddies and a 4x4 grid cell VENT (i.e., h/4) doesn't provide that (plus I don't think we could easily scale it to conserve mass).  

Another possible approach is that a jet fan generally has an axial centrifugal fan inside. This will impose some swirl on the flow. With a 4x4 outlet you could partition it into 4 2x2 outlets, one per quadrant, and then use UVW for HVAC on the VENT. The images show a jet fan with the outlet at the end of the OBST with UVW set to give 45, 22.5, or 0 degree tangential flows in each quadrant towards the adjacent quadrant.  This would not work with h/2 as tangential flow needs at least two adjacent grid cells with the same tangent to function and h/2 in quadrants would be one cell per quadrant.
   
Smokeview images show velocity averaged over 40 s.  In the line plots the B and K curves are the Baturin and Kummel correlations from https://support.thunderheadeng.com/tutorials/pyrosim/modeling-jet-fans/

![image](https://github.com/user-attachments/assets/6853d2c8-5f3b-4e4f-be1e-d4422aaeeb68)

![image](https://github.com/user-attachments/assets/cc0b53fd-ea50-4474-a14e-541858d12673)

**Reply by Z-0815** [reply]
@drjfloyd 
Thank you for your support, and happy New Year!
I’ve started a CFD simulation with an angle of 22.5° and hope I’ve adjusted the input file correctly.
I have a general question: due to the high air induction, a continuous flow eventually develops in the room, and the backflow at the sides disappears.
What is your general opinion on this? Do you also have suggestions on how to best perform a throw distance analysis?

**Reply by drjfloyd** [reply]
@Z-0815 Running the cases I plotted above is probably the most work I have done specifically looking at jet fans. So not much experience to draw on. In an enclosed tunnel not surprised with the results you are describing in terms of the bulk flow overcoming the sunction from upstream at the inlet. I'd look to try and find actual data for jet fan velocity measurements downstream See what it takes in terms of grid size vs. jet fan size and flow to give you decent results and then apply those lessons to your specific case.

**Comment by Er9y714** [comment]
Have you compared FDS vs Fluent for a jet fan flow in the open air (not flush with a wall)? The contribution of the wall boundary might be observed by comparing the results of that and the wall runs (current post).

**Reply by Z-0815** [reply]
I currently do not have access to Fluent to perform a direct comparison, but I have a CFD report that includes Fluent simulations of jet fan flows. My goal is to replicate these results or achieve comparable outcomes using FDS. I am focusing on accurately modeling the jet behavior, particularly the velocity decay and flow characteristics downstream, and would greatly appreciate any insights or suggestions on how to approach this.

**Reply by drjfloyd** [reply]
How do you know the result in the paper are accurate? Do they compare against actual data?

**Reply by Z-0815** [reply]
I don’t have the input data from the paper, so I can’t compare the results in detail. However, I have observed that the results for throw distance and the jet stream in the paper are distinctly different from my own results obtained using FDS. While the fan data align well, the jet stream in my FDS simulations does not disperse as expected. This discrepancy might be due to an issue with my application of FDS, but I have not been able to identify the source of the error. For this reason, I am seeking assistance to better understand and resolve this matter.
